src/certifier.mak

```diff
@@ -55,22 +55,42 @@ LINK=g++
 # Point this to the right place, if you have to, based on your machine's install:
 # PROTO=/usr/local/bin/protoc
 PROTO=protoc
+
+# protoc --python_out will generate this file
+PROTOC_PB2_PY = certifier_pb2.py
```

Comment:
many of the changes here are like this one, introducing symbols to refactor repeatedly used names of targets, files etc.